qa/rpc-tests/p2p-versionbits-warning.py

```diff
@@ -14,11 +14,14 @@
 Test version bits' warning system.
 
 Generate chains with block versions that appear to be signalling unknown
-soft-forks, and test that warning alerts are generated.
+forks, and test that warning alerts are generated.
 '''
 
-VB_PERIOD = 144 # versionbits period length for regtest
-VB_THRESHOLD = 108 # versionbits activation threshold for regtest
+# bip135 begin
+# modified from 108/144 to 50/100 for new unknown versions algo
```

Comment:
Because the "unknown versions" checker can no longer rely on 108/144 being the activation threshold of a deployment (the numbers here apply to regtestnet, but the argument applies generally). It is why I had to replace the unknown versions checker code entirely with something more general, and I picked 51/100 as the new threshold at which to warn the user, as that seemed the minimal majority.